undocked/tests/sample/ext/drv/sample_ext.c

```diff
@@ -74,7 +74,9 @@ static ebpf_program_data_t _sample_ebpf_extension_program_data = {
     .program_type_specific_helper_function_addresses = &_sample_ebpf_extension_helper_function_address_table,
     .global_helper_function_addresses = &_sample_global_helper_function_address_table,
     .context_create = &_sample_context_create,
-    .context_destroy = &_sample_context_destroy};
+    .context_destroy = &_sample_context_destroy,
+    .maximum_irql = PASSIVE_LEVEL,
```

Comment:
why? We should have this as DISPATCH too IMO.

Reply:
OK, sure.